Core/HLE/sceMp4.cpp

```diff
@@ -455,8 +465,15 @@ const HLEFunction sceAac[] = {
 	{0XD2DA2BBA, &WrapU_U<sceAacResetPlayPosition>,              "sceAacResetPlayPosition",           'x', "x"      },
 };
 
+const HLEFunction mp4msv[] = {
+	{0x3C2183C7, &WrapU_UU<mp4msv_3C2183C7>,                    "mp4msv_3C2183C7",               'x', "xx"      },
+	{0x9CA13D1A, &WrapU_UU<mp4msv_9CA13D1A>,                    "mp4msv_9CA13D1A",               'x', "xx"      },
+
+};
+
 void Register_sceMp4()
 {
 	RegisterModule("sceMp4", ARRAY_SIZE(sceMp4), sceMp4);
 	RegisterModule("sceAac", ARRAY_SIZE(sceAac), sceAac);
-}
+	RegisterModule("mp4msv", ARRAY_SIZE(mp4msv), mp4msv);
```

Comment:
Please double check that old save states still work.

-[Unknown]

Reply:
Fixed